tools/windows-node-installer/test/e2e/clustersetup.go

```diff
@@ -0,0 +1,195 @@
+package e2e
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	authscheme "github.com/openshift/client-go/authorization/clientset/versioned/scheme"
+	clientset "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/openshift/windows-machine-config-bootstrapper/tools/windows-node-installer/pkg/client"
+	corev1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	coreclient "k8s.io/client-go/kubernetes/typed/core/v1"
+	rbacclient "k8s.io/client-go/kubernetes/typed/rbac/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/auth"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	// configDir is a directory containing all yaml files for the resources
+	configDir = "clusterconfig"
+	// projectName represents namespace where the resources should be created
+	projectName = "windows-node-installer"
+	// resourceName represents name of the resources to be created
+	resourceName = "windows-node-installer"
+)
+
+// configFiles contains name of the resource files
+var configFiles = [...]string{"namespace.yaml", "role.yaml", "clusterole.yaml", "service_account.yaml", "cluster_role_binding.yaml", "role_binding.yaml"}
```

Comment:
Nit: instead of using yaml files, you could have created these entities in the code using structs and created them programmatically using client-go. Please note that this is a nit and you don't need to address this.